octokit/client.py

```diff
@@ -9,11 +9,13 @@
 
 # https://code.google.com/p/uri-templates/wiki/Implementations
 
-from .resources import Resource
+from resources import Resource
```

Comment:
Woah, that's weird.. and you are able to run everything else? The __init__.py file also has this